examples/react/start-basic-ory/src/routes/_authenticated.tsx

```
@@ -0,0 +1,12 @@
+import { createFileRoute, Outlet, redirect } from '@tanstack/react-router'
```

ESLint

[error] Member 'Outlet' of the import declaration should be sorted alphabetically.

(sort-imports)